components/Board/Board.jsx

```diff
@@ -0,0 +1,28 @@
+import styles from '@/styles/Board.module.css'
+import Card from '@/components/Board/Card'
+import Line from '@/components/Board/Line'
+
+const Board = (props) => {
+    const { image, info } = props
+    return (
+        <div className={styles.container}>
+            <div>
+                <h3 className={styles.title}>{'Our Board'}</h3>
+            </div>
+            <div className={styles.board_box}>
+                <div className={styles.card_one}>
+                    <Line />
+                    <Card image={image.marcelo} info={info.marcelo} />
```

Comment:
Please use an array of objects for the cards info

Reply:
Done. Thanks for the sandbox!